packages/sync-actions/src/products.js

```diff
@@ -129,6 +130,9 @@ function createProductMapActions(
       )
     )
 
+    if (publish === true)
+      return flatten(allActions).map((action) => ({ ...action, staged: false }))
```

Comment:
Will this also work for the [setKey](https://docs.commercetools.com/http-api-projects-products#set-key) and [setTaxCategory](https://docs.commercetools.com/http-api-projects-products#set-taxcategory) actions? 

I see they are in the list here
https://github.com/commercetools/nodejs/blob/2008434e12e469d4d035ed4658cff51069ba7985/packages/sync-actions/src/product-actions.js#L26
and here
https://github.com/commercetools/nodejs/blob/2008434e12e469d4d035ed4658cff51069ba7985/packages/sync-actions/src/product-actions.js#L36

but am not sure how it works.

Reply:
Thats good question, i thought its only `setkey` that doesnt have this flag and i already tested it and it worked, seems that platform ignored  this flag, i should also try it with `setTaxCategory `

Reply:
Sounds good to me. And we don't have integration tests, right? So I guess it's all up to us to manually test it and make sure the platform doesn't throw on us.

Reply:
yep so far we dont have integration tests, and i just tried it with `setTaxCategory` and it worked fine